Formula/dooit.rb

```diff
@@ -0,0 +1,62 @@
+class Dooit < Formula
+  include Language::Python::Virtualenv
+
+  desc "TUI todo manager"
+  homepage "https://github.com/kraanzu/dooit"
+  url "https://files.pythonhosted.org/packages/b1/c6/cc549b6f4223bbc90e80007b69edb8c590788ba1eeaf74f9e1f44cd061c7/dooit-0.1.0.tar.gz"
+  sha256 "05ffbac0e6711276edcae4ca8c211e210081b6f3de0af3d3abafc392834731ad"
+  license "MIT"
+  head "https://github.com/kraanzu/dooit.git", branch: "main"
+
+  depends_on "poetry" => :build
+  depends_on "python@3.10"
+
+  uses_from_macos "expect" => :test
+
+  resource "commonmark" do
+    url "https://files.pythonhosted.org/packages/60/48/a60f593447e8f0894ebb7f6e6c1f25dafc5e89c5879fdc9360ae93ff83f0/commonmark-0.9.1.tar.gz"
+    sha256 "452f9dc859be7f06631ddcb328b6919c67984aca654e5fefb3914d54691aed60"
+  end
+
+  resource "Pygments" do
+    url "https://files.pythonhosted.org/packages/59/0f/eb10576eb73b5857bc22610cdfc59e424ced4004fe7132c8f2af2cc168d3/Pygments-2.12.0.tar.gz"
+    sha256 "5eb116118f9612ff1ee89ac96437bb6b49e8f04d8a13b514ba26f620208e26eb"
+  end
+
+  resource "pyperclip" do
+    url "https://files.pythonhosted.org/packages/a7/2c/4c64579f847bd5d539803c8b909e54ba087a79d01bb3aba433a95879a6c5/pyperclip-1.8.2.tar.gz"
+    sha256 "105254a8b04934f0bc84e9c24eb360a591aaf6535c9def5f29d92af107a9bf57"
+  end
+
+  resource "rich" do
+    url "https://files.pythonhosted.org/packages/f5/f3/f87be42279b5cfba09f7f29e2f4a77063ccf5d9075042981e2cf48752d51/rich-12.4.4.tar.gz"
+    sha256 "4c586de507202505346f3e32d1363eb9ed6932f0c2f63184dea88983ff4971e2"
+  end
+
+  resource "textual" do
+    url "https://files.pythonhosted.org/packages/8c/d1/c228993e8a21e24bb43a0376b2901b6f3f2033dae13e7f76d1103bb9b8a3/textual-0.1.18.tar.gz"
+    sha256 "b2883f8ed291de58b9aa73de6d24bbaae0174687487458a4eb2a7c188a2acf23"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install resources
+    poetry = Formula["poetry"].opt_bin/"poetry"
+    system poetry, "build", "--format", "wheel", "--verbose", "--no-interaction"
+    venv.pip_install_and_link Dir["dist/dooit-*.whl"].first
+  end
+
+  test do
+    # Test based on https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/jrnl.rb
+    script = (testpath/"script.exp")
+    script.write <<~EOS
+      #!/usr/bin/expect -f
```

Comment:
```suggestion
      #!/usr/bin/env expect -f
```

Reply:
A `#!/usr/bin/env` shebang won't work for running commands with flags unless the `-S` flag is used, which isn't supported on all systems. This shouldn't matter, though, because the shebang isn't actually what runs the script; it's the `system "expect", "-f", "script.exp"` line.

I'll make the changes with the `-S` flag. Tell me if you just want me to remove the shebang altogether.